src/debugging/netSdk/NetSdkDebugHelper.ts

```diff
@@ -37,7 +36,7 @@ export class NetSdkDebugHelper extends NetCoreDebugHelper {
 
         await netContainerBuild(netCoreBuildContext);
         if (netCoreBuildContext?.containerBuildOptions === AllNetContainerBuildOptions[1]) {
-            const appProjectAbsolutePath = await this.inferProjPath(context.actionContext, context.folder);
+            const appProjectAbsolutePath = await this.inferProjPath(context, options);
```

Comment:
Consider removing options from the arguments to inferProjPath and use it like:

```suggestion
            const appProjectAbsolutePath = options?.appProject || await this.inferProjPath(context);
```

Reply:
I see, that looks clean